.circleci/config.yml

```diff
@@ -0,0 +1,69 @@
+version: 2.1
+
+executors:
+  my-executor:
+    docker:
+      - image: cimg/go:1.15.3
+    working_directory: ~/sparagn-media-service
+    environment:
+      PROJECT_NAME: sparagn-media-service
+      TERM: dumb
+      SSH_USERNAME: sparagn
+
+jobs:
+  build:
+    executor: my-executor
+    steps:
+      - checkout
+      - run:
+          name: decrypting with git-secret
+          command: |
+            sudo apt-get update
+            sudo apt-get install git-secret -y
+            echo -e $GPG_KEY_CI > ./private_key.gpg
+            gpg --import ./private_key.gpg
+            git-secret reveal
+      - run: go clean && go build
```

Comment:
Is the go build run in Dockerfile ? We may run a build and test here like: `go test ./...`

Reply:
we could have done another job just called test with that. Maybe in another PR?

Reply:
ok, let's do for another PR. I added in the backlog: https://trello.com/c/R5shjuBp